README.md

```diff
@@ -137,5 +139,22 @@ Certificates verified successfully!
 Metadata Signature Verified successfully!
 ```
 
+#### Example 2
```

Comment:
should we rename from Example 2(heading) to Example to verify premium packages? and similarly rename example 1

Reply:
done.